doc/source/admin/conda_faq.rst

```diff
@@ -148,39 +145,73 @@ be configured in the ``dependency_resolvers_conf.xml`` and the ``galaxy.ini`` fi
 #. Automatically at tool run time - When a tool is run and a dependency
    is not found, Galaxy will attempt to install the dependency using
    Conda if ``conda_auto_install`` is activated in the configuration.
-#. Via the API , upon tool installation (>= 16.07) - The Galaxy
-   community maintains an `ansible role`_ that uses
-   BioBlend_ and the Galaxy API to install tools.
+#. Via the API (>= 16.07) - The Galaxy community maintains an `ansible role`_
+   that uses BioBlend_ and the Galaxy API to install tools.
 
 
 3. What is required to make use of this? Any specific packages, Galaxy revision, OS version, etc.?
 *********************************************************************************
 
 The minimum required version of Galaxy to use Conda is 16.01, however
 version 16.07 or greater is recommended. The 16.07 release of Galaxy has
-a nice graphical user interface to manage packages, but this is not
+a graphical user interface to manage packages, but this is not
 required to have Conda dependencies managed and used by Galaxy.
 
 Conda packages should work on all compatible operating systems with
-glibc version (2.5) or newer (this includes Centos 5). We will most
-likely switch soon to glibc version 2.12 as minimum requirement (this
+*glibc* version 2.5 or newer (this includes Centos 5). We will most
+likely switch soon to *glibc* version 2.12 as a minimum requirement (this
 includes CentOS 6). So all packages will run on all \*nix operating
 systems newer than 2007.
 
 
-4. How do I know what system is being used by a given tool?
+4. If I have Conda enabled, what do I need to do to install tools using it? For example, how can I install the latest Trinity? And how will I know the dependencies are installed?
 *********************************************************************************
 
-The Galaxy log will tell you which dependency resolution system is used
+This depends on your ``galaxy.ini`` setting. Galaxy will install the Conda
+package manager for you if you have enabled ``conda_auto_init``. If
+``conda_auto_install`` is enabled, Galaxy will install Trinity via Conda only
+when a Trinity job is launched and Trinity is not yet installed. With release
+16.07 you can see which dependencies are being used in the “Manage installed
+tools” section of the Admin panel and you can select whether or not to install
+Conda packages or Tool Shed package recipes when you install new tools there,
+even if ``conda_auto_install`` is disabled.
```

Comment:
This is a good a question and a fine answer - but I wonder if the answer should note that you can install dependencies along with tools right when installing a tool.

"In 16.07, out of the box when installing new tools you will be given the option to attempt installs of conda packages as well. ..."  Sort of moving the last part of this paragraph first. Not a big thing though.